méli-mélo/2024-04-stepsquiz/index.html

```diff
@@ -0,0 +1,152 @@
+						<li class="radio">
+							<input type="radio" name="role" id="nok-1">
+							<label for="nok-1">Next of Kin</label>
+						</li>
+						<li class="radio">
+							<input type="radio" name="role" id="friend-1">
+							<label for="friend-1">A Friend</label>
+						</li>
+						<li class="radio">
+							<input type="radio" name="role" id="rep-1">
+							<label for="rep-1">Legal Rep</label>
+						</li>
+						<li class="radio">
+							<input type="radio" name="role" id="exec-1">
+							<label for="exec-1">Executor</label>
+						</li>
+					</ul>
+				</div>
+			</fieldset>
+			<fieldset>
+				<legend>2. Age of the deceased?</legend>
+				<div>
+					<p>Confirm how old was the deceased person at the time of death:</p>
+					<ul class="list-unstyled lst-spcd-2 gc-chckbxrdio">
+						<li class="radio">
+							<input type="radio" name="age" id="017-1">
+							<label for="017-1">0-17</label>
+						</li>
+						<li class="radio">
+							<input type="radio" name="age" id="1859-1">
+							<label for="1859-1">18-59</label>
+						</li>
+						<li class="radio">
+							<input type="radio" name="age" id="60-1">
+							<label for="60-1">60+</label>
+						</li>
+					</ul>
+				</div>
+			</fieldset>
+			<fieldset>
+				<legend>3. Location of the deceased?</legend>
+				<div>
+					<p>Specify if the death occur inside Canada or outside Canada:</p>
+					<ul class="list-unstyled lst-spcd-2 gc-chckbxrdio">
+						<li class="radio">
+							<input type="radio" name="location" id="incanada-1">
+							<label for="incanada-1">Inside Canada</label>
+						</li>
+						<li class="radio">
+							<input type="radio" name="location" id="outcanada-1">
+							<label for="outcanada-1">Outside Canada</label>
+						</li>
+					</ul>
+				</div>
+			</fieldset>
+			<fieldset>
+				<legend>4. Miscellaneous</legend>
+				<div>
+					<p>Select all that apply to the deceased:</p>
+					<ul class="list-unstyled lst-spcd-2 gc-chckbxrdio">
+						<li class="checkbox">
+							<input type="checkbox" id="tfw1">
+							<label for="tfw1">Temporary Foreign Worker</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="rc1">
+							<label for="rc1">Resident of Canada</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="goc1">
+							<label for="goc1">Public Service (Employees of the GoC)</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="caf1">
+							<label for="caf1">Canadian Armed Forces</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="rcmp1">
+							<label for="rcmp1">RCMP</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="student1">
+							<label for="student1">Student</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="fn1">
+							<label for="fn1">First Nations/Meti/Inuit</label>
+						</li>
+					</ul>
+				</div>
+			</fieldset>
```

Comment:
Do you need a name and a value for this button? Unless there is another button with name/value pair, use an input[type=hidden] instead if you really need that name/value pair set on that submit button.